docker-java/src/test/java/com/github/dockerjava/cmd/AttachContainerCmdIT.java

```diff
@@ -213,8 +222,6 @@ public void onNext(Frame frame) {
             .withStdIn(stdin)
             .exec(callback);
 
-        dockerClient.startContainerCmd(container.getId()).exec();
```

Comment:
I think this line should stay. Otherwise, the completion won't happen

Reply:
This test is about `attachContainerCmd()` is not supported for Jersey. And the callback should be updated with the error right after executing the method. So it seems that `start` is not required.

Reply:
yet, IMO the test should follow the normal path before getting the exception

Reply:
However... I think it is true the test is complete with `startContainerCmd()`.